ethpm/backends/registry.py

```diff
@@ -33,7 +33,10 @@
 
 # TODO: Update registry ABI once ERC is finalized.
 REGISTRY_ABI = fetch_standard_registry_abi()
-RegistryURI = namedtuple("RegistryURI", ["address", "chain_id", "name", "version", "ens"])
+RegistryURI = namedtuple(
+    "RegistryURI",
+    ["address", "chain_id", "name", "version", "namespaced_asset", "ens"]
```

Comment:
FYI, if you aren't aware, what you are calling `namespace_asset` is a [JSON Pointer](https://tools.ietf.org/html/rfc6901)

Reply:
Oh, good to know! I'm not sure if it's strictly a JSON pointer? Essentially, when trying to specify a certain deployment - I'm trying to avoid requiring the entire blockchain URI. Do you think that's a bad idea? 

Basically...
`ethpm://maker.ethpm.eth:1/dai@1.0.0/deployments/DeploymentName` 
Is what I'm aiming for. . . . It seems like there might be some edge cases where this isn't ideal - in terms of forked chains that share a chain ID? Since this is for the widget - I was thinking of just specifying that a registry URI with a pointer/namespaced asset only supports the 5 main chains.